torch/_export/utils.py

```diff
@@ -42,6 +41,50 @@
 }
 
 
+def _get_shape_env(gm):
```

Comment:
type annotation, maybe also change the name to '_get_shape_env_from_graph_module' or something?

Reply:
Soudns good!